docs/Public_API.rst

```diff
@@ -1123,12 +1123,22 @@ A JSON payload is required to emit your own metrics with the metric value and th
 
         "name": "<CUSTOM METRIC NAME>",
 
-        "value": <CUSTOM METRIC VALUE>
+        "value": <CUSTOM METRIC VALUE>,
+
+        "unit": "<CUSTOM METRIC UNIT>"
 
       }]
 
   }
 
+<INSTANCE INDEX> is the index of current application instance which would be fetched from environment variable `CF_INSTANCE_INDEX`
+
+<CUSTOM METRIC NAME> is the name of the emit metric which must be equal to the metric name that you define in the policy file. 
+
+<CUSTOM METRIC VALUE> is value that you would like to submit.
```

Comment:
The value will be marshaled to a float value. So the "CUSTOM METRIC VALUE" must be a number.